style.css

```diff
@@ -0,0 +1,170 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  position: relative;
+}
+
+.body {
+  width: 100vw;
+  min-height: 100vh;
+  position: relative;
+}
+
+header {
+  position: sticky;
+  top: 0;
+}
+
+nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 30px;
```

Comment:
Good job with the navbar. But let's stick to the design and give the same amount of padding like on the design.

<img width="1019" alt="Screenshot 2023-01-16 at 11 14 15 PM" src="https://user-images.githubusercontent.com/25887253/212759328-e0ab2b09-1f36-4de5-a2cc-eeb2591fd544.png">
<img width="495" alt="Screenshot 2023-01-16 at 11 14 53 PM" src="https://user-images.githubusercontent.com/25887253/212759333-1c5ae061-d1a9-408e-92b4-bd73d027d667.png">